DTAdmin/RootNavController.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  RootNavController.swift
+//  DTAdmin
+//
+//  Created by mac6 on 10/18/17.
+//  Copyright © 2017 if-ios-077. All rights reserved.
+//
+
+import UIKit
+
+class RootNavController: UINavigationController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        //temp variable
+        let isLogined = true
+        
+        if !isLogined {
```

Comment:
Why do we need this?

Reply:
I will get bool value from NSUserDefaults in variable isLogined, and if user doesn't logged, then sent it to login screen

Reply:
I prefer to see working code. Let's remove this and add it back when we will have part with getting value from NSUserDefaults